apps/src/applab/manageAssets.jsx

```diff
@@ -0,0 +1,138 @@
+var React = require('react');
+var AssetsApi = require('./manageAssets/clientApi');
+var AssetRow = require('./manageAssets/assetRow.jsx');
```

Comment:
I've been naming my jsx files that export a component with an uppercase first letter. Might be worth being consistent about this.